### PR TITLE
subsys: net: ip: Support ping times

### PR DESCRIPTION
**Pull request by bublover**
It's convenient to ping for times.
Usage:
ping host [times]
For example: ping 192.168.1.1 10, this will trigger ping for 10 times.

Signed-off-by: Bub Wei <bub.wei@unisoc.com>

### CODE REVIEW

**Review by easonxiang — APPROVED** [review]

File: subsys/net/ip/net_shell.c [hunk 1]
```diff
@@ -2799,26 +2800,32 @@ static int cmd_net_ping(const struct shell *shell, size_t argc, char *argv[])
...
 
-	if (IS_ENABLED(CONFIG_NET_IPV6)) {
-		ret = ping_ipv6(shell, host);
-		if (!ret) {
-			goto wait_reply;
-		} else if (ret == -EIO) {
-			PR_WARNING("Cannot send IPv6 ping\n");
-			return -ENOEXEC;
-		}
+	if (argv[2]) {
```
Comment (easonxiang) [inline subsys/net/ip/net_shell.c]: I'm not sure if argv[i] are set to NULL by default, if not, we need check argc as well. 
if ((argc > 2) && argv[2])

**Review by easonxiang — CHANGES_REQUESTED** [review]

**Comment by jukkar** [comment]
Please check #14024 which is providing already lot of improvements for ping command.

File: subsys/net/ip/net_shell.c (hunk 1 above)
Comment (bublover) [reply subsys/net/ip/net_shell.c]: Just refer to argv[1]

Comment (bublover) [reply subsys/net/ip/net_shell.c]: Just refer to argv[1]

Comment (bublover) [reply subsys/net/ip/net_shell.c]: Thanks @easonxiang.
Just refer to argv [1] above.

Well.We might follow another PR Jukkar memtioned.

**Comment by bublover** [comment]
> Please check #14024 which is providing already lot of improvements for ping command.

Sure, it looks nice.

**Pull request closed without merging**